pennylane/_version.py

```diff
@@ -16,4 +16,4 @@
 Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.10.0-dev"
+__version__ = "0.10.0"
```

Comment:
So this should not be 0.11.0-dev now, since we start merging PRs soon?

Reply:
As soon as this PR is merged, we can make a GitHub release + tag + upload to pypi. Then, the next commit is to bump to 0.11.0-dev, and the feature freeze is complete 🙂